main.go

```diff
@@ -111,7 +112,7 @@ func InitFlags(fs *pflag.FlagSet) {
 		"The address the health endpoint binds to.")
 
 	// Add --feature-gates flag
-	featuregate.DefaultMutableFeatureGate.AddFlag(fs)
+	feature.MutableGates.AddFlag(fs)
```

Comment:
Does this also need to be added to `bootstrap/kubeadm/main.go`? If it's not, I suspect the feature gate checks added in any of the `bootstrap/*` wouldn't have any effect.

Reply:
Answered above, yeah I don't wanna rush anything out, I need to take some time to think about this a little more